cpp/src/ray/runtime/task/task_executor.cc

```diff
@@ -148,18 +148,9 @@ Status TaskExecutor::ExecuteTask(
   std::shared_ptr<msgpack::sbuffer> data = nullptr;
   ArgsBufferList ray_args_buffer;
   for (size_t i = 0; i < args_buffer.size(); i++) {
-    auto &ref = arg_refs.at(i);
```

Comment:
Can we also remove `arg_refs ` from args of this function? Seems it is unused after your change.

Reply:
This function is callback with the given signature required by CoreWorker, other language workers use the data for their own reasons.

Reply:
Got.

Reply:
We need arg_refs to get the object id of arg_ref, some arguments are object ref args not a value arg.